v2/cmd/asoctl/internal/importing/importable_arm_resource.go

```diff
@@ -352,7 +352,7 @@ func (*importableARMResource) classifyError(err error) (string, bool) {
 func (i *importableARMResource) createImportableObjectFromID(
 	owner *genruntime.ResourceReference,
 	armID *arm.ResourceID,
-) (genruntime.ImportableARMResource, error) {
+) (resource genruntime.ImportableARMResource, failure error) {
```

Comment:
minor: Would it make sense to use `err` here rather than failure?

Reply:
I was avoiding a wider rewrite of the method - need to avoid shadowing of `err` with locals. I've done this now.